web/src/features/header/Header.tsx

```diff
@@ -57,10 +59,16 @@ function MenuLink({
 export default function Header(): JSX.Element {
   const isMobileApp = Capacitor.isNativePlatform();
   const setIsFAQModalOpen = useSetAtom(isFAQModalOpenAtom);
-  const onFAQClick = () => {
-    setIsFAQModalOpen(true);
-  };
   const { t } = useTranslation();
+
+  const onFAQClick = useCallback(() => {
+    setIsFAQModalOpen(true);
+  }, [setIsFAQModalOpen]);
+
+  const handleClick = useCallback(() => {
+    trackEvent(TrackEvent.HEADER_LINK_CLICKED, { linkId: 'get-data' });
+  }, []);
```

Comment:
I'll move this outside the function instead as it don't need to be inside the component due to static data.